File: JavaProjects/Clase6276-3-HerenciaMultiple/src/ParentClasses/Mamifero.java
package ParentClasses;

public class Mamifero {
    // Propiedades
    private String nombre;
    // Métodos
    // Constructor
    public Mamifero(String nombre) {
        this.nombre = nombre;
        System.out.println("\nHa creado un mamífero llamado " + nombre);
    }
    // Método propio de la clase Mamifero
    public void amamantar(){
        System.out.println("Tomo leche proveniente de mi madre.");
    }
    // Getter y Setter
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: C++Projects/6276_cppclase1-2_archivosdistintos/Mamifero.cpp
#include "Mamifero.h"

// Constructor de la clase
Mamifero::Mamifero(string nombre){
    this->nombre = nombre;
}

// Getter y Setter
string Mamifero::getNombre(){
    return nombre;
}
void Mamifero::setNombre(string nombre){
    this->nombre = nombre;
}
